questions.ts

```diff
@@ -670,7 +670,7 @@ export const askQuestions = async (
   }
 
   // Ask for LlamaCloud API key when using a LlamaCloud index or LlamaParse
-  if (isUsingLlamaCloud || program.useLlamaParse) {
+  if ((isUsingLlamaCloud || program.useLlamaParse) && !program.llamaCloudKey) {
```

Comment:
shouldn't `isUsingLlamaCloud` already be true, because of:
```
 program.dataSources = [
    {
      type: "llamacloud",
      config: {},
    },
    EXAMPLE_FILE,
  ];
```
above?